src/token/handler.ts

```diff
@@ -355,10 +363,6 @@ export class Login7TokenHandler extends TokenHandler {
   onPacketSizeChange(token: PacketSizeEnvChangeToken) {
     this.connection.messageIo.packetSize(token.newValue);
   }
-
-  onDatabaseMirroringPartner(token: DatabaseMirroringPartnerEnvChangeToken) {
-    // Do nothing
-  }
 }
```

Comment:
I moved this handler to be right below the `onDatabaseChange` event handler for better ordering and to keep the event next to all of the other `ENVCHANGE` events.